recommender/utils.py

```diff
@@ -154,3 +156,40 @@ def gen_json_dict(panel_id: str, anonymous_user: bool = False) -> Dict[str, Any]
         json_dict["user_id"] = User.objects.first().id
 
     return json_dict
+
+
+def load_examples() -> Dict:
+    """
+    If possible, generates examples for /recommendations request using
+     appropriate database.
+    If not, generates artificial examples.
+
+    Returns:
+        examples: Examples dict.
+    """
+
+    examples = {
+        "categories": [1, 2, 3],
+        "geographical_availabilities": [1, 2, 3],
+        "providers": [1, 2, 3],
+        "related_platforms": [1, 2, 3],
+        "scientific_domains": [1, 2, 3],
+        "target_users": [1, 2, 3],
+    }
+
+    if os.environ["FLASK_ENV"] == "testing":
```

Comment:
I would add a comment on why we don't use current_app.config - it's a special case

Reply:
bo tego tam nie ma? XD

Reply:
nie

Reply:
no nie ma